unison-src/transcripts/add-test-watch-roundtrip.output.md

```diff
@@ -0,0 +1,21 @@
+```unison
+test> foo : [Test.Result]
+foo = []
+```
+
+Apparently when we add a test watch, we add a type annotation to it, even if it already has one. We don't want this to happen though!
+
+```ucm
+.> add
+
+  ⍟ I've added these definitions:
+  
+    foo : [Result]
+
+.> view foo
+
+  foo : [Result]
+  foo : [Result]
```

Comment:
This is still showing the incorrect output, isn't it?

Reply:
Confusing; sorry.
Yes, but that transcript doesn't involve `update`. So it's tracked by loose end #4417.

Reply:
Oh I see the "we don't want this to happen" is saying that this is a bug but the transcript is just capturing that it currently happens, right?

Reply:
Right. I don't think you will actually run into #4417 unless you are manually putting type signatures on your tests as in that transcript though. Let me know if you find otherwise!